config/config/common-config.yaml

```diff
@@ -118,6 +118,11 @@ falco:
       enabled: false
       version: 2.0.0
 
+  artifact:
+    install:
+      # set to false in an air-gapped environment, unless artifacts are self-hosted
+      enabled: true
```

Comment:
Do we really want this on by default even for regular flavours?
What benefit does it bring compared to the rules bundled with the current image, is there a current version skew?

Reply:
I am not sure if there is any real benefit for having it enabled, I just kept it enabled from previous config. I can maybe try and look into if there is any benefit in keeping it enabled

Reply:
Currently we seem to get falco rules for a specific version [since this PR](https://github.com/elastisys/compliantkubernetes-apps/pull/1843/files) when artifact `install` is enabled (I noticed [v3.0.0](https://github.com/falcosecurity/rules/releases/tag/falco-rules-3.0.0) of falco rules dropped recently).  However, I feel like we can keep it like it is now and maybe look deeper into this in a separate issue, I created [this issue](https://github.com/elastisys/compliantkubernetes-apps/issues/1669) long ago before `artifact.install` was enabled in apps, so maybe it can be investigated as part of that issue instead.